src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -0,0 +1,476 @@
+                interContigPair.serialize(kryo, output);
+            }
+
+            @Override
+            public InterContigPair read( final Kryo kryo, final Input input, final Class<InterContigPair> klass ) {
+                return new InterContigPair(kryo, input);
+            }
+        }
+    }
+
+    @DefaultSerializer(OutiesPair.Serializer.class)
+    public static final class OutiesPair extends BreakpointEvidence {
+        OutiesPair( final GATKRead read, final ReadMetadata metadata ) {
+            super(read, metadata);
+        }
+
+        private OutiesPair( final Kryo kryo, final Input input ) { super(kryo, input); }
+
+        @Override
+        public boolean equals( final Object obj ) {
+            return obj instanceof OutiesPair && compareTo((OutiesPair)obj) == 0;
+        }
+
+        @Override
+        public String toString() {
+            return super.toString() + " OutiesPair";
+        }
+
+        public static final class Serializer extends com.esotericsoftware.kryo.Serializer<OutiesPair> {
+            @Override
+            public void write( final Kryo kryo, final Output output, final OutiesPair mateUnmapped ) {
+                mateUnmapped.serialize(kryo, output);
+            }
+
+            @Override
+            public OutiesPair read( final Kryo kryo, final Input input, final Class<OutiesPair> klass ) {
+                return new OutiesPair(kryo, input);
+            }
+        }
+    }
+
+    @DefaultSerializer(SameStrandPair.Serializer.class)
+    public static final class SameStrandPair extends BreakpointEvidence {
+        SameStrandPair( final GATKRead read, final ReadMetadata metadata ) {
+            super(read, metadata);
+        }
+
+        private SameStrandPair( final Kryo kryo, final Input input ) { super(kryo, input); }
+
+        @Override
+        public boolean equals( final Object obj ) {
+            return obj instanceof SameStrandPair && compareTo((SameStrandPair)obj) == 0;
+        }
+
+        @Override
+        public String toString() {
+            return super.toString() + " SameStrandPair";
+        }
+
+        public static final class Serializer extends com.esotericsoftware.kryo.Serializer<SameStrandPair> {
+            @Override
+            public void write( final Kryo kryo, final Output output, final SameStrandPair sameStrandPair ) {
+                sameStrandPair.serialize(kryo, output);
+            }
+
+            @Override
+            public SameStrandPair read( final Kryo kryo, final Input input, final Class<SameStrandPair> klass ) {
+                return new SameStrandPair(kryo, input);
+            }
+        }
+    }
+
+    @DefaultSerializer(WeirdTemplateSize.Serializer.class)
+    public static final class WeirdTemplateSize extends BreakpointEvidence {
+        private final int templateSize;
+
+        WeirdTemplateSize( final GATKRead read, final ReadMetadata metadata ) {
```

Comment:
Yes.  The information was a dilute soup, and despite efforts to only pick up extremes, the template size evidence was contributing too many intervals all by itself.  We can turn this code back on if we put a weight on BreakpointEvidence and give these guys a lesser weight, or put a rule in place that prohibits calling intervals that are supported only by template size evidence, or both.  Need something a little more sophisticated to pull this in.  (Or maybe it's not needed.)